packages/core/src/accordion/accordion.stories.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import '@clr/core/accordion/register.js';
+import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs } from '@clr/core/internal';
+import { action } from '@storybook/addon-actions';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Components/Accordion/Stories',
+  component: 'cds-accordion',
+  argTypes: getElementStorybookArgTypes('cds-accordion-panel', customElements),
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=1007%3A0',
+    },
+  },
+};
+
+export const API = (args: any) => {
+  return html`
+    <cds-accordion>
+      <cds-accordion-panel
+        ...="${spreadProps(getElementStorybookArgs(args))}"
+        @expandedChange=${action('expandedChange')}
+      >
+        <cds-accordion-header>${args['cds-accordion-header']}</cds-accordion-header>
+        <cds-accordion-content>${args['cds-accordion-content']}</cds-accordion-content>
+      </cds-accordion-panel>
+    </cds-accordion>
+  `;
+};
+
+export const basicAccordion = () => {
+  return html`
+    <cds-accordion>
+      <cds-accordion-panel expanded>
+        <cds-accordion-header>Item 1</cds-accordion-header>
+        <cds-accordion-content>Content 2</cds-accordion-content>
+      </cds-accordion-panel>
+      <cds-accordion-panel disabled>
+        <cds-accordion-header>Item 2</cds-accordion-header>
+        <cds-accordion-content>Content 2</cds-accordion-content>
+      </cds-accordion-panel>
+      <cds-accordion-panel>
+        <cds-accordion-header>Item 3</cds-accordion-header>
+        <cds-accordion-content>Content 3</cds-accordion-content>
+      </cds-accordion-panel>
+    </cds-accordion>
+  `;
+};
+
```

Comment:
As discussed, we'll incorporate this into the pattern work since this is more of a "how to use this component in an application" rather than component's API.

Reply:
Because we have no place to put "patterns" and because we have no current plan in place as to how we are tackling patterns, documenting them, or storing them for users to get to, I'm concerned that this is pushing a common ask/use case into the indefinite future. I feel like we're going to get asked this and have to answer it over and over. 

I've erred on the side of including a "pattern" or two in the stories for now (see: accessible circular progress, nested/layered overlays, etc.) so that users have working examples now. Given the limitations of storybook, there's a bit of a leap between the story and what a user would implement. But the story demonstrates that something is possible and the raw mechanics of how it would be accomplished.

Reply:
I think something like this will work in storybook

```html
<cds-accordion>
  <cds-accordion-panel expanded>
    <cds-accordion-header>Header 1</cds-accordion-header>
    <cds-accordion-content>Content 1</cds-accordion-content>
  </cds-accordion-panel>
  <cds-accordion-panel id="async-panel">
    <cds-accordion-header>Header 1</cds-accordion-header>
    <cds-accordion-content>
      
    </cds-accordion-content>
  </cds-accordion-panel>
</cds-accordion>
<script>
  const asyncPanel = document.querySelector('#async-panel');
  const asyncPanelContent = asyncPanel.querySelector('cds-accordion-content');

  asyncPanel.addEventListener('expandedChange', () => {
    asyncPanel.expanded = !asyncPanel.expanded;

    if (asyncPanel.expanded) {
      asyncPanelContent.innerHTML = `<cds-progress-circle></cds-progress-circle>`;
      setTimeout(() => {
        asyncPanelContent.innerHTML = `<p>hello world</p>`;
      }, 3000);
    } else {
      // reset for demo
      asyncPanelContent.innerHTML = `<cds-progress-circle></cds-progress-circle>`;
    }
  });
</script>
```

Reply:
I had a hard time tracking this down but I see the example in `accordion-panel.stories.ts`

Thanks for doing that!